docs/blueprints/index.md

```diff
@@ -70,13 +70,13 @@ Components to which upstreaming doesn't apply were left out from the table.
 
 | Component | Upstream                        | Qubes OS
 | :-------: | :------:                        | :------:
-| GRUB2     | [Sent][grub]                    | [Yes][grub-qos]
+| GRUB2     | [Sent][grub-up]                 | [Yes][grub-qos]
```

Comment:
Was this discovered manually or by some tool? I wonder if CI could help with such issues somehow.

Reply:
Manually.  Seems like something a markdown linter could find, but don't know if they have such check.